src/modules/vtol_att_control/standard.cpp

```diff
@@ -60,7 +60,7 @@ Standard::Standard(VtolAttitudeControl *attc) :
 	_mc_yaw_weight = 1.0f;
 	_mc_throttle_weight = 1.0f;
 
-	_params_handles_standard.pusher_ramp_dt = param_find("VT_PSHER_RMP_DT");
+	_params_handles_standard.pusher_ramp_dt = param_find("VT_PUSHER_RMP_DT");
```

Comment:
@RomanBapst @sanderux You OK with the param name changes here? AFAIK parameters can be max 16 chars, and this is IMO better.

Reply:
"VT_PUSHER_RMP_DT" - there are exactly 16 characters

Reply:
Yes @EgoPingvina - I checked and I am happy with that, but I'm not maintainer of this code - hence query to @RomanBapst